packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -354,7 +414,12 @@ export class BlockList extends Component {
 							this.addBlockToEndOfPost( paragraphBlock );
 						} }
 					>
-						<View style={ styles.blockListFooter } />
+						<View
+							style={ [
+								styles.blockListFooter,
```

Comment:
It would be ideal to only apply this offset when the inserter is opened and animate it to avoid a jump, but I would imagine that might be difficult to accomplish. I wanted to provide the thought nonetheless. Maybe an opportunity for future enhancement.